### PR TITLE
add support for k8s PriorityClass

### PR DESCRIPTION
**Pull request by lancespeelmon**

### CODE REVIEW

File: elasticsearch/values.yaml [hunk 1]
```diff
@@ -92,6 +92,10 @@ extraInitContainers: []
   #   image: busybox
   #   command: ['do', 'something']
 
+# This is the PriorityClass settings as defined in
+# https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass
+priorityClassName: {}
```
Comment (Crazybus) [inline elasticsearch/values.yaml]: I think this should be an empty string looking at https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#pod-priority.

**Comment by lancespeelmon** [comment]
@Crazybus I think I have addressed all your concerns. Thanks, L

**Comment by lancespeelmon** [comment]
ping

**Comment by Crazybus** [comment]
jenkins test this please

**Pull request merged**